packages/shared/src/components/modals/UserListModal.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { ReactElement, useRef, useState } from 'react';
+import { UserShortInfoPlaceholder } from '../profile/UserShortInfoPlaceholder';
+import { Modal, ModalProps } from './common/Modal';
+import UserList, { UserListProps } from '../profile/UserList';
+import { InfiniteScrollingProps } from '../containers/InfiniteScrolling';
+import { UserShortProfile } from '../../lib/user';
+
+export interface UserListModalProps
+  extends Omit<ModalProps, 'children'>,
+    Pick<UserListProps, 'additionalContent'> {
+  users: UserShortProfile[];
+  placeholderAmount?: number;
+  title: string;
+  scrollingProps: Omit<InfiniteScrollingProps, 'children'>;
+}
+
+function UserListModal({
+  users,
+  title,
+  scrollingProps,
+  placeholderAmount,
+  additionalContent,
+  ...props
+}: UserListModalProps): ReactElement {
+  const container = useRef<HTMLElement>();
+  const [modalRef, setModalRef] = useState<HTMLElement>();
+
+  return (
+    <Modal
+      contentRef={(e) => setModalRef(e)}
+      kind={Modal.Kind.FlexibleCenter}
+      size={Modal.Size.Small}
```

Comment:
Should it not accept this as props now?

Reply:
The spread operator below overrides this property since the prop extended from `ModalProps`.

Reply:
Oh, wait. Yep, removing this will be better and make it come from the props.

Reply:
Yeah accept it from props, but maybe default to Small or something like that.